Source/WebKit/UIProcess/mac/WebViewImpl.mm

```diff
@@ -3132,25 +3132,49 @@ static String commandNameForSelector(SEL selector)
     m_page->capitalizeWord();
 }
 
+#if USE(APPLE_INTERNAL_SDK)
+#include <WebKitAdditions/WebViewImplAdditions.mm>
+#else
+static Vector<WebCore::CompositionUnderline> extractInitialUnderlines(NSAttributedString *string)
```

Comment:
Nit - I would either omit the `string` parameter here, or add an `UNUSED_PARAM(string)` in the implementation.